client/coral-embed-stream/src/Embed.js

```diff
@@ -114,6 +123,11 @@ class Embed extends Component {
     }
   }
 
+  handleClick() {
```

Comment:
You can just do `handleClick = () => {` to have it automatically bound to `this`.